.travis.yml

```diff
@@ -8,10 +8,14 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  - gas -fmt=json -out=results.json ./... > /dev/null
+  - cat ./results.json
+  - test $(grep -c "\"severity\":\s*\"HIGH\"" ./results.json) -lt 8
```

Comment:
What's this 8? Can u pls provide some sample output and make code reviewers lives easier perhaps? ;-)